src/markdown.c

```diff
@@ -1403,7 +1403,7 @@ prefix_quote(uint8_t *data, size_t size)
 	if (i < size && data[i] == ' ') i++;
 	if (i < size && data[i] == ' ') i++;
 
-	if (i < size && data[i] == '>') {
+	if ((i < size && data[i] == '>') && (i + 1 < size && data[i+1] != '!')) {
```

Comment:
also the `> ` (blank space) case seems to be unchecked for spoiler-blockquotes? should there be a `data[i+2] == ' '` equivalent of the check on 1407 for in spoilerquotes? (i'm not super clear on what that case does)